pkg-manager/headless/package.json

```diff
@@ -84,6 +84,8 @@
     "@pnpm/modules-yaml": "workspace:*",
     "@pnpm/package-is-installable": "workspace:*",
     "@pnpm/package-requester": "workspace:*",
+    "@pnpm/patching.config": "workspace:*",
+    "@pnpm/patching.types": "workspace:*",
```

Comment:
I believe you can import PatchFile from lockfile.fs, no need to add a new dependency.

Reply:
https://github.com/pnpm/pnpm/pull/8337/commits/2e8937d722d63b11e9330a855120fd3f4a2f1d54